core/tx_pool.go

```diff
@@ -1032,8 +1032,8 @@ func (pool *TxPool) runReorg(done chan struct{}, reset *txpoolResetRequest, dirt
 				delete(events, addr)
 			}
 		}
-		// Reset needs promote for all addresses
-		promoteAddrs = promoteAddrs[:0]
+		// make a capable slice to avoid modify accountSet cache.
```

Comment:
The old comment was referring to the logical flow. Please add that back. The new comments seems to be just a text-explanation of the line below, which doesn't make much sense. Please revert that.

Reply:
Fixed